### DISCUSSION TITLE
An automatic fish catcher

### DISCUSSION THREAD

**Discussion by Thepigcat76** [TechReborn/TechReborn#3053] [Ideas]
A single-block machine (fishing net) to catch fish.

This would also allow for some really cool automations like sponges, methane and even mercury to encourage fishing. The machine should be a bit random output wise. 

Outputs: deep oceans = treasures
                river and ocean biome = fish
                oceans = pufferfish and tropical fish
                Tropical regions = bamboo and 
                increased chance for tropical- and 
                pufferfish

Gui:         3*2 inventory, no upgrades

Thank you to ayutac, spearkiller and vonzato for helping me design this idea!

**Comment by VonZato** [comment]
Please
